src/main/java/net/openhft/chronicle/core/io/Wget.java

```diff
@@ -0,0 +1,46 @@
+package net.openhft.chronicle.core.io;
+
+import net.openhft.chronicle.core.pool.StringBuilderPool;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.net.URL;
+
+import static net.openhft.chronicle.core.io.Closeable.*;
+
+public final class Wget {
+
+    private static StringBuilderPool stringBuilderPool = new StringBuilderPool();
+
+    private Wget() {
+    }
+
+    /**
+     * performs an http get
+     *
+     * @param url the url of the http get
+     * @return the result, as a string
+     * @throws IOException if the connection could not be established
+     */
+    public static CharSequence url(String url) throws IOException {
```

Comment:
I think it would be better to provide a StringBuilder rather than magically get a TL on each invocation.  Maybe I pass the returned object around and then overwrite it on the next invocation.